shell/components/formatter/InternalExternalIP.vue

```diff
@@ -38,21 +38,19 @@ export default {
       -
     </template>
     /
-    <template>
```

Comment:
took a quick look, i think this fixes https://github.com/rancher/dashboard/issues/11819? cc @mantis-toboggan-md

Reply:
Will leave any follow up for that PR